src/engine/SCons/Action.py

```diff
@@ -73,7 +73,7 @@
 """
 
 #
-# __COPYRIGHT__
+# Copyright (c) 2001, 2002, 2003, 2004, 2005, 2006, 2007 The SCons Foundation
```

Comment:
Rob:

   This is NOT a change made in my copy. It is exactly what is in the current archive. This change, and the change to version number with your ID on it, are not present in the GIT repo. They are only present in the archive (and, of course, the checked out copy of the archive we are all using). These two changes are in-effective, of course. They don’t do anything. But they are indicative that we are not using the GIT repo directly.

                                                                   Michael

From: Rob Boehne [mailto:notifications@github.com]
Sent: Thursday, August 04, 2016 3:18 PM
To: datalogics/scons
Cc: Michael McKeough; Author
Subject: Re: [datalogics/scons] Update the powershell change to make it apply to C# as well as Java. (#4)


In src/engine/SCons/Action.py<https://github.com/datalogics/scons/pull/4#discussion_r73594725>:

> @@ -73,7 +73,7 @@

>  """

>

>  #

> -# __COPYRIGHT__

> +# Copyright (c) 2001, 2002, 2003, 2004, 2005, 2006, 2007 The SCons Foundation

Somehow you've expanded these macros. If you can use a simple text editor to undo these two changes, please fix this.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/datalogics/scons/pull/4/files/8cf99517f944f32793f9e6528e7653135e018980#r73594725>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABQVhf4LkY72nuEz-IR5FbbsJsgQ_t-4ks5qckjogaJpZM4JdEVh>.